src/hotspot/share/memory/metaspaceShared.cpp

```diff
@@ -73,6 +73,8 @@
 #include "gc/g1/g1CollectedHeap.inline.hpp"
 #endif
 
+#include <string.h>
```

Comment:
This include is strange, usually we do not include std head file in .cpp file.

Reply:
The include is needed for strcasecmp and _stricmp.
If you do a grep like `find . -name "*.cpp" | xargs grep "#include \<"`, you'll find many .cpp files include std header files.